torchvision/models/feature_extraction.py

```diff
@@ -172,8 +175,25 @@ def _warn_graph_differences(train_tracer: NodePathTracer, eval_tracer: NodePathT
     warnings.warn(msg + suggestion_msg)
 
 
+def _get_leaf_modules_for_ops():
+    members = inspect.getmembers(torchvision.ops)
+    result = []
+    for _, obj in members:
+        if inspect.isclass(obj) and issubclass(obj, torch.nn.Module):
+            result.append(obj)
+    return tuple(result)
+
+
 def get_graph_node_names(
-    model: nn.Module, tracer_kwargs: Dict = {}, suppress_diff_warning: bool = False
+    model: nn.Module,
+    tracer_kwargs: Dict = {
+        "autowrap_modules": (
+            math,
+            torchvision.ops,
+        ),
+        "leaf_modules": _get_leaf_modules_for_ops(),
+    },
```

Comment:
If you define the default value like that, it's going to be mutated for all future calls. Moreover the method calls to get the values won't happen on the first call but on the method declaration.

It's best to instantiate with something non-mutable, like `None` for instance and then if the value is still None, set the default within the method. This idiom is very common in TorchVision and is mainly done for such cases.